src/style.css

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
```suggestion
```

Reply:
fit it https://github.com/mate-academy/layout_stars/pull/2357/files#r1066189879